module/x/gravity/keeper/keeper.go

```diff
@@ -67,7 +67,6 @@ func NewKeeper(
 		PowerReduction: powerReduction,
 	}
 	k.EthereumEventProcessor = EthereumEventProcessor{
-		keeper:     k,
 		bankKeeper: bankKeeper,
 	}
```

Comment:
I don't really understand why is this hardcoded... what's the point of having the interface if it's always going to be the same? This should be a constructor parameter

Reply:
I just changed it into a keeper method.